spec/classes/snmp_client_spec.rb

```diff
@@ -51,9 +51,8 @@
             mode: '0644',
             owner: 'root',
             group: 'root',
-            path: '/etc/snmp/snmp.conf',
-            require: ['Package[snmp-client]', 'File[/etc/snmp]']
-          )
+            path: '/etc/snmp/snmp.conf'
```

Comment:
I'd be happy with trailing commas. Much smaller diff :)

Reply:
rubocop won't be though :(

Reply:
We may have to teach rubocop a lesson then. There's an option for it.

Reply:
meh.  Just think of the noise correcting all 100 odd modules to the new style! ;)

Reply:
thats what the rubocop:autocorrect task is for? :)